### PR TITLE
Dimension search with select-2 component 

### PR DESCRIPTION
**Pull request by pjain1**
Adding ability to filter dimension by doing a search

### CODE REVIEW

File: app/controllers/datasource/show.js [hunk 1]
```diff
@@ -191,6 +188,44 @@ var TimeseriesObject = Ember.ObjectProxy.extend(
...
+        this.query
+      ).then(
+        function (val)
+        {
+          var retVal = { data:[] };
+          if (val.length > 0) {
+            retVal.data = val.map(
+              function (res) {
+                return res.result.map(
+                  function (result) {
```
Comment (mike-north) [inline app/controllers/datasource/show.js]: es6 arrow function

File: app/controllers/datasource/show.js [hunk 2]
```diff
@@ -191,6 +188,44 @@ var TimeseriesObject = Ember.ObjectProxy.extend(
...
+          dataSource: this.datasource
+        },
+        this.query
+      ).then(
+        function (val)
+        {
+          var retVal = { data:[] };
+          if (val.length > 0) {
+            retVal.data = val.map(
+              function (res) {
```
Comment (mike-north) [inline app/controllers/datasource/show.js]: es6 arrow function

File: package.json [hunk 3]
```diff
@@ -19,9 +19,9 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.0.2",
-    "ember-cli": "0.2.1",
+    "ember-cli": "1.13.1",
```
Comment (mike-north) [inline package.json]: may as well upgrade to 1.13.7

File: package.json [hunk 4]
```diff
@@ -35,6 +35,7 @@
     "ember-data-model-fragments": "^0.3.1",
```
Comment (mike-north) [inline package.json]: we should consider replacing model fragments with the new ember data stuff (embedded records mixin)

File: app/controllers/datasource/show.js (hunk 2 above)
Comment (pjain1) [inline app/controllers/datasource/show.js]: sure, will change

File: app/controllers/datasource/show.js (hunk 1 above)
Comment (pjain1) [inline app/controllers/datasource/show.js]: ok

File: package.json (hunk 4 above)
Comment (pjain1) [inline package.json]: OK..let me see

File: package.json (hunk 3 above)
Comment (pjain1) [inline package.json]: ok

Comment (pjain1) [inline package.json]: Upgrading error - "version not found: ember-cli@1.13.7". However 1.13.6 works fine.

File: package.json (hunk 4 above)
Comment (pjain1) [inline package.json]: can we just skip this change in this PR as I guess going ahead jj-abrams will be the main branch.

**Comment by mike-north** [comment]
I'll take a look at the CD pipeline for this project. Looks like the build is failing because of some sauce labs problem

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by pjain1** [comment]
@mike-north sure..

**Comment by pjain1** [comment]
@mike-north any updates on this PR ?

**Pull request merged**